### DISCUSSION TITLE
Migration guide from v0

### DISCUSSION THREAD

**Discussion by josesimoes** [Live-Charts/LiveCharts2#1244] [General]
Hello,

Any plans for a migration guide from v0 apps?
Something simple with the common patterns, differences, would be great to get things started.

BTW, great work you've doing with this! Thank you. 😃 👍🏻

**Comment by beto-rodriguez** [comment]
Hello!

There are a lot of differences, v0 used the WPF API, now well we handle a lot of things we use SkiaSharp to render everything, it is complicated to list all the differences, the API of v2 is based on v0, but there are differences.

The main difference is (I think) that now we can use any `IEnumerable` as the source of data, in v0, we were forced to use the `ChartValues<T>` class.
